### ISSUE TITLE
Provide aliases with flags for playbook tags and `just` "recipes"

### ISSUE THREAD

**Issue by luixxiul** [spantaleev/matrix-docker-ansible-deploy#3865]
**Is your feature request related to a problem? Please describe.**
It would be nice if `install-all` and `setup-all` could be available via `i -a` or `s -a`, in the sense that `npm install --global` can be shortened to `npm i -g`.

For reference, `npm-install` provides these aliases: `add, i, in, ins, inst, insta, instal, isnt, isnta, isntal, isntall` according to its manual.

**Describe the solution you'd like**
See above.

**Describe alternatives you've considered**

**Additional context**
To make it compatible with other tags like `setup-SERVICE`,  we would probably need to investigate what kind of scenarios need to be covered.

**Comment by aine-etke** [comment]
`just` is not NPM, neither is `justfile`. While that technically possible to implement such approach, in reality it will require horrible `if { if { .. } }` quirks that will just make it unreadable and unmaintainable.

Here is an existing example (yes, it doesn't reflect exactly what the topic requests, but still could be considered as a demonstration) for `just update` with agru (reaction to different flags): https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/justfile#L17

**Comment by luixxiul** [comment]
> `just` is not NPM, neither is `justfile`.

I would like to point out just in case that I picked up NPM as an example for adopting flags and aliases.

**Comment by aine-etke** [comment]
Sorry, to avoid any confusion let me rephrase: `just` is yet-another `make` alternative, and `justfile` is a "cleaner `Makefile`".

So, while a regular program (like NPM from your example) can accept various flags and sub-commands, including shorthanded, `just`'s job is to work with recipes in a form of `just do-somethig`. While it's technically possible to write recipes (and an alias will still be a recipe, just wrapping the original one) like `just i -a`, that looks controversial to the whole idea of recipe files.

As an alternative, why not use bash aliases, like `alias jia="just install-all"`?

**Comment by luixxiul** [comment]
Thanks for clarification! Using aliases indeed seems reasonable :+1:

Closing as wontfix.
